frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -75,15 +73,13 @@ local symbol_prefix = {
         mem_usage = C_("FooterLetterPrefix", "M:"),
         -- @translators This is the footer letter prefix for Wi-Fi status.
         wifi_status = C_("FooterLetterPrefix", "W:"),
-        -- no prefix for custom text
-        custom_text = "",
     },
     icons = {
         time = "⌚",
         pages_left_book = BD.mirroredUILayout() and "↢" or "↣",
         pages_left = BD.mirroredUILayout() and "⇐" or "⇒",
         battery = "",
-        bookmark_count = "☆",
+        bookmark_count = "\u{F097}",
```

Comment:
May be add a comment: `-- bookmark symbol, from our nerdfont` (if I guessed right).